Library/Homebrew/formula.rb

```diff
@@ -936,6 +936,12 @@ def plist
     nil
   end
 
+  # This method can be overridden to provide a service definition
+  sig { returns(Hash) }
+  def service
```

Comment:
Accepting a block i.e. `service do` would be preferable to overriding the method.